### PR TITLE
Revert Facebook Pixel

### LINKED ISSUE #8390: FaceBook Pixel isn't firing correctly - delete it
From jmcook (06faccc5-5f69-453c-9a9d-0eeed865985c) in the Report a Bug guild:

"**It looks like the FaceBook Pixel isn't firing correctly. I'm using the 'Facebook Pixel Helper' Chrome extension. This doesn't affect anyone, except the companies marketing team.**"

I haven't done anything to verify or investigate this.

FYI: definition of [The Facebook Pixel](https://en-gb.facebook.com/business/help/651294705016616)

### PR DESCRIPTION
**Pull request by librarianmage**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #8390 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

@SabreCat I have a second attempt to revert the files from the commit mentioned in my last PR

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

**Comment by librarianmage** [comment]
Ping!

Ping!

**Comment by paglias** [comment]
LGTM, sorry for the delay

File: website/client-old/js/services/analyticsServices.js [hunk 1]
```diff
@@ -127,10 +107,6 @@
       properties.Level = user.stats.lvl;
       properties.Mana = Math.floor(user.stats.mp);
     }
-
-    properties.balance = user.balance;
-    properties.balanceGemAmount = properties.balance * 4;
```
Comment (SabreCat) [inline website/client-old/js/services/analyticsServices.js:132]: Why were these removed? @MathWhiz @paglias

Comment (paglias) [reply website/client-old/js/services/analyticsServices.js:132]: Uh-oh I hadn't noticed that

**Comment by librarianmage** [comment]
I just reverted the commit, and moved the files into the new structure, so
I have no idea.

**Comment by SabreCat** [comment]
Yeah, looks like in the process you grabbed and reverted some unrelated changes. Third try? :cat::sweat_drops:

**Comment by paglias** [comment]
I've reverted this PR. @MathWhiz probably the commit included some other unrelated changes
